packages/design/src/config/view/FormSummary/index.tsx

```diff
@@ -9,7 +9,8 @@ const FormSummary: FormElementComponent<Pattern<FormSummaryPattern>> = ({
   return (
     <>
       <div className="usa-legend-wrapper">
-        <legend className="usa-legend">{pattern.title}</legend>
+        {/* <legend className="usa-legend">{pattern.title}</legend> */}
+        <h1>{pattern.title}</h1>
```

Comment:
Probably not the most idiomatic, uswds-wise, but looks good for the demo. Can revisit next week.